test/rql_test/connections/asyncio_connection_3_5.py3_5.test

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import os, sys, subprocess
+
+try:
+    import asyncio
+except ImportError:
+    # remove this when implimented https://github.com/rethinkdb/rethinkdb/issues/4139
+    sys.stderr.write('This test requires the asyncio module from Python 3.5')
+    sys.exit(0)
+
+sys.exit(subprocess.call([os.environ.get('INTERPRETER_PATH', 'python'), os.path.join(os.path.dirname(__file__), 'asyncio_connection.py')]))
```

Comment:
It looks like this is launching the other `asyncio` tests, not the file you've added in this PR.